File: app/src/main/java/com/dlka/lastfmnewtracknotifier/Recenttracks.java

package com.dlka.lastfmnewtracknotifier;

import java.util.List;
import com.squareup.moshi.Json;

public class Recenttracks {

    @Json(name = "@attr")
    private Attr attr;
    @Json(name = "track")
    private List<Track> track = null;

    public Attr getAttr() {
        return attr;
    }

    public void setAttr(Attr attr) {
        this.attr = attr;
    }

    public List<Track> getTrack() {
        return track;
    }

    public void setTrack(List<Track> track) {
        this.track = track;
    }

}

File: app/src/main/java/com/dlka/lastfmnewtracknotifier/Recenttracks.kt
package com.dlka.lastfmnewtracknotifier

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class Recenttracks (
    @Json(name = "@attr")
    var attr: Attr? = null,
    @Json(name = "track")
    var track: List<Track>? = null
)